### DISCUSSION TITLE
Changing the Nuxt localhost to be my local ip make the site work only at the first load, then "Unable to fetch collections" happen.

### DISCUSSION THREAD

**Discussion by MohammadAbbas2** [nuxt/nuxt#10260] [Questions (Nuxt 2)]
I want to run Nuxt at my local IP so that I can access the website from any device in my Network, So I run the following command:

`HOST=0 PORT=8000 npm run dev`

The Result:
The website is accessible from any device in my network, But it's working with many data unfetched. It's loaded the first time, but after executing any action, a message appears that says "unable to fetch collections". 

I'll appreciate your help all of you. Thanks in advance!

**Comment by javiercani** [comment]
Could you solve this issue? i'm having same problem.

**Reply by quroom** [reply]
`npm run dev -- --host 127.0.0.1`
127.0.0.1 can be changed whatever you want.
